src/main/java/com/ciandt/techgallery/service/impl/TechnologyServiceImpl.java

```diff
@@ -166,17 +166,27 @@ private void validateInformations(Technology technology) throws BadRequestExcept
    */
   @Override
   public Response getTechnologies() throws InternalServerErrorException, NotFoundException {
-    List<Technology> techEntities = technologyDAO.findAllActiveTechnologies();
+    List<Technology> techEntities = technologyDAO.findAllActives();
     // if list is null, return a not found exception
     if (techEntities == null) {
       throw new NotFoundException(ValidationMessageEnums.NO_TECHNOLOGY_WAS_FOUND.message());
     } else {
       TechnologiesResponse response = new TechnologiesResponse();
+      sortTechnologiesDefault(techEntities);
       response.setTechnologies(techEntities);
       return response;
     }
   }
 
+  private void sortTechnologiesDefault(List<Technology> techEntities) {
```

Comment:
@felipeaziz Please create a class to sort technologies. Sorting technology is not a TechnologyService's responsability. Doing this, the service implementation will be cleaner.